tests/test_sourceSelector.py

```diff
@@ -329,6 +330,18 @@ def testUnresolved(self):
         self.config.unresolved.maximum = None
         self.check((starGalaxy > minimum).tolist())
 
+    def testFiniteRaDec(self):
```

Comment:
Please add the shortest of docstrings, i.e., "Test that non-finite RA and Dec values are caught."